specs/Standards/Tokens/FungibleTokenCore.md

```diff
@@ -0,0 +1,356 @@
+- To prevent the deployed contract from being modified or deleted, it should not have any access keys on its account.
+
+**Interface**:
+
+```javascript
+/************************************/
+/* CHANGE METHODS on fungible token */
+/************************************/
+// Simple transfer to a receiver. Does not call a method.
+// Requirements:
+// * Caller of the method must attach a deposit of 1 yoctoⓃ for security purposes
+// * Caller must have greater than or equal to the `amount` being requested
+// `receiver_id` is the valid NEAR account receiving the fungible tokens.
+// `amount` is the number of tokens to transfer, wrapped in quotes and treated like a string, although the number will be stored as an unsigned integer with 128 bits.
+// The `memo` argument is optional. It's added for use cases that may benefit from indexing or providing information for a transfer.
+function ft_transfer(
+    receiver_id: string,
+    amount: string,
+    memo: string|null
+): void {}
+
+// Transfer tokens and call a method on a receiver contract. A successful workflow will end in a success execution outcome to the callback on the same contract at the method `ft_resolve_transfer`.
+//
+// You can think of this as being similar to attaching native NEAR tokens to a function call on some contract. It allows you to attach any Fungible Token in a call to a receiver contract, given by `receiver_id`, where `msg` gets passed to the receiver contract to indicate both a function to call and the parameters to that function.
+// Requirements:
+// * Caller of the method must attach a deposit of 1 yoctoⓃ for security purposes
+// * Caller must have greater than or equal to the `amount` being requested
+// * The receiving contract must implement `ft_on_transfer` according to the standard.
+// * This contract must implement `ft_resolve_transfer` according to the standard.
+// `msg` is an argument that may specify any information expected by the receiving contract in order to properly handle the function. If the receiving contract needs something larger than a single string, `msg` could be a complex JSON structure, base64-encoded.
+function ft_transfer_call(
+   receiver_id: string,
+   amount: string,
+   memo: string|null,
+   msg: string,
+): Promise {}
+
+// This function isn't called directly and shall implement logic ensuring it's called by "itself" as a callback to the promise sent to the receiver contract.
+// Here the `sender_id` will be the sender from the `ft_transfer_call` method call. The `receiver_id` and `amount` will be the same as the values provided to `ft_transfer_call`.
+// This returns a string representing a string version of an unsigned 128-bit integer of how many tokens were returned. (This value may be different than `amount`.)
+function ft_resolve_transfer(
+   sender_id: string,
+   receiver_id: string,
+   amount: string
+): string {}
```

Comment:
This doesn't have to be part of the main interface. Let's move it below the main interface in a suggestion section

Reply:
As in, the behavior is required, but the function name is not part of the standard?

Reply:
Updated in https://github.com/near/NEPs/pull/158/commits/9b420e671d86c93119b581ef5e41b359b9262710